graylog2-web-interface/src/hooks/useScopePermissions.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import { useQuery } from 'react-query';
+
+import UserNotification from 'util/UserNotification';
+import fetchScopePermissions from 'hooks/api/fetchScopePermissions';
+
+interface ScopeParams {
+  is_mutable: boolean;
+}
+
+type ScopeName = 'DEFAULT' | 'ILLUMINATE';
+
+type EntityScopeRecord = Record<ScopeName, ScopeParams>;
+
+type EntityScopeType = {
+  entity_scopes: EntityScopeRecord,
+};
+
+const useGetPermissionsByScope = () => {
+  const { data, isLoading, isError, error } = useQuery<EntityScopeType, Error>(
+    'scope-permissions',
```

Comment:
Change to array to comply with the new useQuery Signature that is getting pushed in.